hello/validPassthrough.ts

```diff
@@ -1,12 +1,20 @@
 import type { Static } from '@sinclair/typebox'
 import type { ErrorObject } from 'ajv'
 import type { HelloMessage } from './HelloMessage.js'
+import { HistoricalDataRequest } from './chart/HistoricalDataRequest.js'
+import { HistoricalDataResponse } from './chart/HistoricalDataResponse.js'
+import { ProblemDetail } from './errors/ProblemDetail.js'
 import { validator } from './validator.js'
 
 export const validPassthrough = (
 	v: unknown,
 	onDropped?: (v: unknown, errors: ErrorObject[]) => unknown,
-): Static<typeof HelloMessage> | null => {
+):
+	| Static<typeof HelloMessage>
+	| Static<typeof HistoricalDataRequest>
```

Comment:
Only responses should be added here.

Reply:
If we include only responses, then we need to remove types from `validator` function as well to make it match with the type. Therefore, we cannot test historical data request in `hello/chart/examples.spec.ts`